apps/demos/Demos/Chat/Overview/jQuery/styles.css

```diff
@@ -0,0 +1,24 @@
+.demo-container {
+  padding: 20px;
```

Comment:
I wrote that padding is unnecessary here because the required padding is already present in the demo environment. You can check it on the demo sitehttps://js.devexpress.com/React/Demos/WidgetsGallery/Demo/Calendar/Overview/MaterialBlueLight/ 

![Uploading image.png…]()

Reply:
This has been fixed in the commit https://github.com/DevExpress/DevExtreme/pull/28363/commits/01a287acf4d7a1d8f2c3a171baab7ecfee47c6ed to obtain the correct reference in the jQuery approach. Please consider this solution in other frameworks